etc/kayobe/ansible/openstack-host-image-upload.yml

```diff
@@ -42,6 +42,21 @@
             disk_format: qcow2
             state: present
             filename: "{{ local_image_path }}"
+            visibility: shared
+          register: image
+
+        - name: Ensure dependencies are installed
```

Comment:
Can this be integrated into the previous pip task [here](https://github.com/stackhpc/stackhpc-kayobe-config/pull/1631/files#diff-bf56b907ae792936e4ff67a5a7a6bee0cae08ebb282903047eebf1ce2ac7bb71R33)?